### PR TITLE
Handles the special case of a where condition on the table from

### PR DESCRIPTION
**Pull request by shaoshiva**
Handles the special case of a condition in a dedicated query (not a join) whose field is on the table "from" by replacing it with the value of the field property on the model "from".

### CODE REVIEW

File: classes/manymany.php [hunk 1]
```diff
@@ -241,8 +241,22 @@ protected function _build_query_where($query, $conditions, $alias_to, $model_fro
...
+			if (!($condition[0] instanceof \Fuel\Core\Database_Expression)) {
+				// Handles the special case of a condition whose field is on the table from
+				// by replacing it with the value of the field property on the $model_from
+				if (\Str::starts_with($condition[0], $this->table_from.'.')) {
+					list(, $field) = explode('.', $condition[0], 2);
+					if (isset($model_from->{$field})) {
+						$condition[0] = \DB::expr(\DB::quote($model_from->{$field}));
+					} else {
+						// Skip if the field property doesn't exist on the $model_from
+						continue;
```
Comment (smeeckaert) [inline classes/manymany.php]: Perhaps it should throw an Exception instead of silently fail. This could help identify typo.

Comment (shaoshiva) [inline classes/manymany.php]: :+1:

**Comment by smeeckaert** [comment]
:+1:

**Comment by jguyomard** [comment]
:+1:

**Pull request merged**